### PR TITLE
added experiences and notification action types 

### PR DESCRIPTION
**Pull request by JorPoon**
# Description

-Experiences and notification action types created. 
-Meetings actiontypes , action creator and reducer updated.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change status

- [x] Complete, but not tested (may need new tests)

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by JorPoon** [comment]
I will be adding experiences reducer and action creator into this pull request too

**Review by tfolbrecht — DISMISSED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,11896 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Estrax) [inline yarn.lock]: please delete this file and add it to `.gitignore`

**Review by Estrax — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (JorPoon) [reply yarn.lock]: ok i did it

**Review by Estrax — APPROVED** [review]

**Review by khamp19 — APPROVED** [review]
fix your defaults

**Pull request merged**